src/mpid/ch4/netmod/ofi/ofi_am_events.h

```diff
@@ -239,7 +239,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_handle_rdma_read(MPIDI_OFI_am_header_t *
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_OFI_HANDLE_RDMA_READ);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_OFI_HANDLE_RDMA_READ);
 
-    int attr = MPIDIG_AM_ATTR__IS_ASYNC | MPIDIG_AM_ATTR__IS_RNDV;
+    int attr = MPIDIG_AM_ATTR__IS_ASYNC | MPIDIG_AM_ATTR__IS_RNDV | MPIDI_OFI_AM_ATTR__RDMA;
     MPIDIG_global.target_msg_cbs[msg_hdr->handler_id] (am_hdr, NULL, 0, attr, &rreq);
```

Comment:
Pass constant rather than variable.

Reply:
I don't think there would be any effect. And even in the case when the compiler can optimize with the constant, it should have the least trouble to see that attr is a constant and propagate the constant.

Reply:
See about, the compiler cannot optimize on the value of a variable. Passing constant is cheaper and offers a chance for optimization.